ansible/Vagrantfile.CentOS7

```diff
@@ -23,6 +23,10 @@ echo "--------------------------------------------------------------------------
 echo "*** '$VAGRANT_MOUNT/playbooks/hosts' file for ansible-playbook contains ***"
 cat $VAGRANT_MOUNT/playbooks/hosts
 echo "---------------------------------------------------------------------------------------"
+# Get IPs of the VM, and output to shared folder
+ifconfig | grep "inet" | awk '{print $2}' | sed 's/addr://'  >> /vagrant/playbooks/AdoptOpenJDK_Unix_Playbook/hosts.tmp
```

Comment:
Does this definitely work ok on all VMs? I would expect this would return at least two values - an external one and `127.0.0.1`

Reply:
Indeed - I didn't think there'd be any harm in generating `hosts.tmp` even if the user wasn't starting the VM through `testScript.sh` too. And the way it picks the highest was lifted straight from the Windows section - you wrote it :-)